README.md

```diff
@@ -127,11 +131,11 @@ Note that if you plan to adopt TypeScript in the entry file, you should change `
 
 You now have the build pipeline correctly set up with TypeScript handling the transpilation. Try bundling the app with the following command and then open `index.html` in a browser,
 
-```
-node ./node_modules/webpack/bin/webpack.js
+```sh
+node ./node_modules/.bin/webpack

```

Comment:
This is a substance change, but I'd maybe suggest `npx webpack` instead (this is pretty much `npx`'s intended use, right)? Since this is likely platform-dependent (as the bin folder contains bat files among other things).

Reply:
Wesley, this will expect users to have NPM 5.2.*, well. The intent in the PR was just to shorten the webpack execution path. What do you think?

Reply:
eh? I don't think you can actually shorten it the way you did then, since `node ./node_modules/.bin/webpack` doesn't work (on any platform), since the `.bin` folder contents are shell scripts, not js:
![image](https://user-images.githubusercontent.com/2932786/36762294-afdf0bf6-1bd7-11e8-8123-d82c7d05d81c.png)

And I think we're trying to avoid something complex like "invoke in whatever way is correct for your shell the script at `./node_modules/.bin/webpack`", which could be just `./node_modules/.bin/webpack` or `& ./node_modules/.bin/webpack` or a number of other things.

Reply:
You're correct (I've native PC matchine @work only). It works with `npx` tool.  I'll add a line with a link to blog post what the npx tool is
Thanks!

Reply:
Done, rebased!